harness/determined/searcher/_search_runner.py

```diff
@@ -119,7 +119,9 @@ def run_experiment(
 
         try:
             while experiment_is_active:
-                time.sleep(5)
+                time.sleep(
+                    2
+                )  # we don't want to call long polling API more often than every 2 seconds.
```

Comment:
I think the wait should only happen if there's an error? Since the request now blocks on getting anything, waiting shouldn't be necessary normally.